packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -224,4 +246,36 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent {
 	public onSortChange(): void {
 		this.sortChanged.emit(this);
 	}
+
+	/**
+	 * Gets a the classes for a specific cell, based on the cellClassName Input and the cellClassNameFn function if it was given.
+	 * @param row - The data object of the row the cell is in.
+	 * @returns The classes for the cell.
+	 */
+	public getCellClassName(row: any): string {
+		const value: any | undefined = this.getRawValue(row);
+		const classNameFnResult: string =
+			(this.cellClassNameFn && typeof this.cellClassNameFn === "function" && this.cellClassNameFn(value, row, this.name)) || "";
+		return `${classNameFnResult} ${this.cellClassName || ""}`;
```

Comment:
Can you write a test for this code? I'm not sure whether this will output what we expect....